tools/torch2onnx.py

```diff
@@ -0,0 +1,127 @@
+import argparse
+import os.path as osp
+import sys
+import warnings
+
+import mmcv
+import torch
+import torch.nn as nn
+from mmcv.runner import load_checkpoint
+
+from mmaction.models import build_model
+
+try:
+    import onnx
+except ImportError:
+    warnings.warn('Please install onnx to support onnx exporting.')
+
+sys.path.append('../')
+
+
+class RecognizerWrapper(nn.Module):
+    """Wrapper that only inferences the part in computation graph."""
+
+    def __init__(self, recognizer):
+        super().__init__()
+        self.recognizer = recognizer
+
+    def forward(self, x):
+        return self.recognizer.forward_dummy(x)
+
+
+class LocalizerWrapper(nn.Module):
+    """Wrapper that only inferences the part in computation graph."""
+
+    def __init__(self, localizer):
+        super().__init__()
+        self.localizer = localizer
+
+    def forward(self, x):
+        return self.localizer._forward(x)
```

Comment:
It seems that `SSN` doesn't have the `_forward()` method.

Reply:
Yeah, that's tricky. Strictly speaking, SSN is just a head not a 'localizer'
; while bsn, bmn are not 'localizer' either but 'proposer'. Since now bmn and bsn are more like a real 'localizer': it can report mAP with UntrimmedNet classification and the ckpts are already released. Let's just skip it and wait for better abstraction and refacting on this part?

Reply:
yeah we can refactor this later

Reply:
Agree.